test/test_ops.py

```diff
@@ -1258,6 +1258,85 @@ def test_giou_jit(self) -> None:
         self._run_jit_test([[0, 0, 100, 100], [0, 0, 50, 50], [200, 200, 300, 300]])
 
 
+class TestDistanceBoxIoU(BoxTestBase):
+    def _target_fn(self) -> Tuple[bool, Callable]:
```

Comment:
A general question to torchvision maintainers
Do we type hint in tests? 
It's not bad to type hint. But it isn't something which we follow either 😅


cc @pmeier @datumbox

Reply:
No, we don't. See https://github.com/pytorch/vision/pull/5563#discussion_r824758253. The reason is that without also checking the tests with `mypy` they might go out of date and that is usually more harmful than not having them at all.

Reply:
I am removing the type hints then, thanks for the link @pmeier. :+1: